src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -351,6 +351,9 @@
   <data name="net_http_unix_invalid_response" xml:space="preserve">
     <value>The server returned an invalid or unrecognized response.</value>
   </data>
+  <data name="net_http_unix_handler_disposed" xml:space="preserve">
+    <value>The handler was disposed of while active operations were in progress.</value>
```

Comment:
fyi.
I wasn't sure if the wording should be "disposed" instead of "disposed of".  But it is correct to have "dispose of".
http://snarkygrammarguide.blogspot.com/2012/11/dispose-vs-dispose-of-grammar-police.html